v2/internal/controllers/crd_appconfiguration_configstore_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-service-operator/v2/api/appconfiguration/v1beta20220501"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// If recording this test, might need to manually purge the old App config: az appconfig purge --name asotest-confstore-fsrajl
+
+func Test_AppConfiguration_ConfigurationStore_CRUD(t *testing.T) {
```

Comment:
This resource has keys too: https://docs.microsoft.com/en-us/rest/api/appconfiguration/stable/configuration-stores/list-keys?tabs=HTTP

Wondering if we should support keys here like we do for storage accounts, etc?